src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+baseUrl=http://18.209.28.58:30000
```

Comment:
Is this IP the real one?

Reply:
Set a fake one and renamed to EastwooUrl

Reply:
I was not concerned about security... just to knwo what parameter must be configured for a load test. Can you document this?